pkg/config/generate.go

```diff
@@ -376,6 +376,17 @@ func (g *simpleGenerator) Generate(cs *api.OpenShiftManagedCluster, template *pl
 		}
 	}
 
+	if len(c.EtcdMetricsPassword) == 0 {
+		if c.EtcdMetricsPassword, err = random.AlphanumericString(10); err != nil {
+			return
+		}
+	}
+	if len(c.EtcdMetricsUsername) == 0 {
```

Comment:
considering we are building generic tlsProxy, I would make it consistent configurable? Otherwise, we will need to duplicate values here (hardcoding and , tlsProxy config where we would set to the same value)

Reply:
ok